app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -135,6 +136,12 @@ class SettingsActivity : DuckDuckGoActivity() {
         _duckPlayerSettingsPlugin.getPlugins()
     }
 
+    @Inject
+    lateinit var _threatProtectionSettingsPlugin: PluginPoint<ThreatProtectionSettingsPlugin>
```

Comment:
We don't need to do this in this PR, but imho the benefit of plugins is to have a section where we can inject views. Here we define a section where we expect to inject one view, but we are still adding a container every time we need a new entry.
We take benefit of having that view inside our module, but we lose the part where we don't need to touch this activity and the layout every time. I think we should make the protections section plugable (but I understand is out of the scope).